python/ray/tune/registry.py

```diff
@@ -55,6 +60,22 @@ def get_registry():
     return _Registry(_default_registry._all_objects)
 
 
+def _to_pinnable(obj):
+    """Converts obj to a form that can be pinned in object store memory.
+
+    Currently only numpy arrays are pinned in memory, if you have a strong
+    reference to the array value.
+    """
+
+    return (obj, np.zeros(1))
+
+
+def _from_pinnable(obj):
+    """Undoes _to_pinnable."""
```

Comment:
This makes it sound like it unpins the object; maybe `Retrieve from _to_pinnable format`?